tests/test_lpass.py

```diff
@@ -4,61 +4,47 @@
 
 
 class TestLPass(unittest.TestCase):
-    @patch('db_facts.lpass.check_output')
-    def test_lpass_field_notes(self, mock_check_output):
-        mock_check_output.return_value = "fakenotes\n".encode("utf-8")
-        out = lpass.lpass_field('my_name', 'notes')
-        mock_check_output.\
-            assert_called_with(['lpass', 'show', '--notes', 'my_name'])
-        assert out == "fakenotes"
```

Comment:
Is there any chance someone (or some script) might use db-facts to try to retrieve notes from an associated password manager entry? I'd lean towards that being a possibility. Instead of deleting this outright, can you have it throw an error saying this command is no longer supported?

Reply:
To my knowledge this is not something that can happen. I investigated all the ways that db-facts calls into `lpass_field` and there are only a few entry points, none of which care about the notes.

maybe this file at one point was intended to be part of a more general-purpose library and that's why the extra support was added, not sure.

I will definitely raise some errors though for unsupported commands